docs/esp8266/general.rst

```diff
@@ -66,8 +66,27 @@ avoid allocating too big container objects (lists, dictionaries) and
 buffers. There is also no full-fledged OS to keep track of resources
 and automatically clean them up, so that's the task of a user/user
 application: please be sure to close open files, sockets, etc. as soon
-as possible after use.
-
+as possible after use. The need to close sockets is particularly critical.
+
+Socket instances remain active until they are explicitly closed. This has two
+consequences. Firstly they occupy RAM, so an application which opens sockets
+without closing them will eventually fail. Secondly an open socket can cause
+unexpected behaviour, notably the issuing of ``LmacRxBlk:1`` errors. This can
+occur after a program raises an exception. This includes a ``KeyboardInterrupt``
+resulting from issuing ctrl-C. The REPL prompt appears, but at a later time
+the socket receives data resulting in failure. This manifests as a repeated
+issuing of ``LmacRxBlk:1`` which continues until the ESP8266 is hard reset.
+
+To prevent this, errors should be trapped as follows::
+
+    try:
```

Comment:
a) The general notice is that documentation is not a bug tracker. Bugs should be in the tracker and be fixed. Documentation should be clear and not provide incorrect information (which it will if it duplicates contents of bug tracker).

b) I don't see how #3208 relates to the discussion. A socket should be closed *regardless* of whether there was a normal program program termination or error. The (easiest) way to achieve that is try/finally, and that's what docs should show as an illustration of the point. Of course, in particular apps other means than try/finally can be used, but that section doesn't teach generic programming principles, it just explains that socket should be always closed, nothing else, and should do that in the most obvious way.